src/transform.rs

```diff
@@ -94,6 +99,15 @@ impl<P: EuclideanSpace, R: Rotation<P>> Transform<P> for Decomposed<P::Diff, R>
     }
 
     #[inline]
+    fn inverse_transform_vector(&self, vec: P::Diff) -> Option<P::Diff> {
+        if ulps_eq!(self.scale, &P::Scalar::zero()) {
```

Comment:
Do we need an `ulps_eq!` here? We're just trying to avoid the `NaN`, right?

Reply:
Interesting, I see we have one on `inverse_transform`. Were you going for consistency with my code? 🤔

Reply:
Yes, I went with the consistency